ThBoxUnitTests/UnitTests.cs

```diff
@@ -647,10 +647,11 @@ public void ForEach_Test()
         [DataRow(2,2,2)]
         [DataRow(0.4,0.03,0.15)]
         public void CompressedBoxVolume_ShouldEqualGivenBoxVolume(double a, double b, double c)
-        {
+        {s
```

Comment:
s? It would be nice to check before pushing. For example in some GUI application like GitKraken or Rider Git.